### PR TITLE
feat: add retros

### LINKED ISSUE #89: Retro | ability to delete items

### LINKED ISSUE #92: Retro | show players in current session

### LINKED ISSUE #93: Retro | Socket disconnected notification
same as poker, we want to know when the socket is disconnected

### PR DESCRIPTION
**Pull request by BearAlliance**
closes #89 
closes #92 
closes #93 

Screen-Recording-2020-04-04-at-1

### CODE REVIEW

File: server/routes/retro-socket.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+          });
+          break;
+        case 'UNLIKE_ITEM':
+          console.log(`Retro ${retro.id}: ${playerId} unliking item ${message.itemId}`);
+          Object.keys(retro.columns).forEach(columnName => {
+            retro.columns[columnName] = retro.columns[columnName].map(item => {
+              if (item.id === message.itemId) {
+                console.log(item);
+                console.log(playerId);
+                item.votes = item.votes.filter(vote => vote.playerId !== playerId);
```
Comment (olore) [inline server/routes/retro-socket.js:76]: this should be enough, shouldn't need to set it in line 72 or return it on 79 
i think ;)

Comment (BearAlliance) [reply server/routes/retro-socket.js:76]: oh true, I started with this by just copying the `LIKE_ITEM` stuff

File: src/pages/game/active-game.jsx [hunk 2]
```diff
@@ -1,9 +1,9 @@
 import React, { Fragment, useEffect, useState } from 'react';
-import { InviteLink } from './invite-link';
+import { InviteLink } from '../../components/invite-link';
 import { Scorecard } from './scorecard';
 import { StoryTitleSection } from './story-title-section';
 import { VotingButtons } from './voting-buttons';
-import { WebSocketClient } from '../../websocket-client';
+import { PokerSocket } from '../../websocket-client';
```
Comment (olore) [inline src/pages/game/active-game.jsx:6]: 😏

**Comment by BearAlliance** [comment]
last push was a rebase to fix the merge conflicts

**Pull request merged**